src/dsn.layer2.thrift

```diff
@@ -45,6 +45,12 @@ enum app_status
     AS_RECALLING
 }
 
+struct duplication_options {
+    1:string app_name;
```

Comment:
Is it neccessary to have `app_name` in `duplication_options`, it seems to be duplicated.